python/StartServer.py

```diff
@@ -27,6 +28,69 @@ def do_exit(self, line):
         """
         shutdown()
 
+    def do_findobjs(self, line):
+        """ findobjs
+        Looks for objects where a given dimension matches a given value for a
+        given set.
+        """
+        tokens = shlex.split(line)
+        if len(tokens) == 3:
+            type_text = tokens[0]
+            dim = tokens[1]
+            val = tokens[2]
+            if type_text in fs.name2class:
+                tp = fs.name2class[type_text]
+                if hasattr(tp, dim):
+                    objs = fs.Store.get(tp)
+                    for obj in objs.values():
+                        v = obj[dim]
+                        if v == val:
```

Comment:
not necessary. If the dimension was an int or float, it would remain int and float. If the dimension was a list, it would be a list.